common/test/acceptance/tests/helpers.py

```diff
@@ -199,6 +199,20 @@ def enable_css_animations(page):
     """)
 
 
+def coordinates_for_scrolling(page, css_selector):
+    """
+    Get the x and y coordinates of elements
+    """
+    coordinates_dict = page.browser.find_element_by_css_selector(css_selector)
```

Comment:
Since find_element_by_css_selector returns an element, coordinates_dict as a variable name is a bit confusing. Maybe rename it "element" or "elem" or something like that?